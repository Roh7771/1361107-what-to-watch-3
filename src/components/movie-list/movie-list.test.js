import React from "react";
import renderer from "react-test-renderer";
import MovieList from "./movie-list";

const mock = {
  filmsList: [
    {
      title: `Any Cool Film`,
      imgSrc: `Some path`,
      id: 5
    },
    {
      title: `Another Cool Film`,
      imgSrc: `Some path`,
      id: 9
    },
  ]
};

it(`<MovieList /> should render correctly`, () => {
  const tree = renderer
    .create(<MovieList filmsList={mock.filmsList} onMovieCardClick={() => {}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
