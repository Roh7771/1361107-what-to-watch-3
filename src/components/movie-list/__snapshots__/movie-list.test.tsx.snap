// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieList /> should render correctly 1`] = `
Array [
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      href="/films/2"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          className=""
          height={175}
          poster="Some Path"
          src="some path"
          width={280}
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/2"
        onClick={[Function]}
      >
        Some Title
      </a>
    </h3>
  </article>,
  <article
    className="small-movie-card catalog__movies-card"
    onClick={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    <a
      href="/films/3"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <video
          className=""
          height={175}
          poster="Some Path"
          src="some path"
          width={280}
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/3"
        onClick={[Function]}
      >
        Some Title
      </a>
    </h3>
  </article>,
]
`;
